zipkin-lens/src/components/App/App.test.js

```diff
@@ -37,34 +38,48 @@ describe('<App />', () => {
     mount.cleanUp();
   });
 
-  it('should update document title', () => {
+  it('should update document title when mounted', () => {
+    // Since it is not possible to verify that document.title has
+    // been changed using shallow rendering, use mount.
     mount(<App />);
     expect(document.title).toBe('Zipkin');
   });
 
-  it('should render providers', () => {
+  it('should provide React Context for MuiPickers', () => {
```

Comment:
close.. I would ask why do I care about MuiPicker? Which function of the app does MuiPicker provide?

Ex 'should provide React Context for time range selection' is more a why MuiPicker is a what